src/apps/dashboard/routes/_legacyRoutes.ts

```diff
@@ -145,5 +145,11 @@ export const LEGACY_ADMIN_ROUTES: LegacyRoute[] = [
             view: 'dashboard/streaming.html',
             controller: 'dashboard/streaming'
         }
+    }, {
+        path: 'playback/trickplay',
```

Comment:
We really shouldn't be adding new pages using the legacy view/controller functionality. Any new code should be preferably written in typescript + react, but given we are wanting this in for the imminent 10.9 release I think we can let it slide.

Reply:
> We really shouldn't be adding new pages using the legacy view/controller functionality. Any new code should be preferably written in typescript + react, but given we are wanting this in for the imminent 10.9 release I think we can let it slide.

Oh, that's my bad. I wrote this a while ago to go alongside the server PR so I never saw the pinned issue about using React on jellyfin-web. I've never used React but I see some of the user dashboard pages have already been converted so I'll take a shot at converting this too.